contents/docs/api/post-only-endpoints.mdx

```diff
@@ -4,164 +4,147 @@ sidebar: Docs
 showTitle: true
 ---
 
-> **Update:** These endpoints can now be accessed with either your Team API key or your [personal API key](/docs/api/overview).
+PostHog provides POST-only public endpoints to capture events, update user or group data, evaluate feature flags and more. These use your [project API key](https://app.posthog.com/project/settings) and do not return any sensitive data from your PostHog instance.
 
-As explained in our [API overview](/docs/api/overview) page, PostHog provides two different APIs.
+## Sending events
 
-This page refers to our public endpoints, which use the same API key as the [PostHog snippet](/docs/integrate/client/js). The endpoints documented here are used solely with `POST` requests, and will not return any sensitive data from your PostHog instance.
+PostHog is event-based meaning your analytics data takes the form of a value sent by a specific user with optional addition data. Much of the functionality of PostHog, including updating users or groups, combining users, and evaluating feature flags, is driven by this event-based structure. [Our SDKs](/docs/libraries) handle the different types of events for you, but with the API, you need to send the right type of event (listed below) to get the functionality you want. The default PostHog events have the `$` prefix. 
 
-> **Note:** For this API, you should use your 'Project API Key' from the 'Project' page in PostHog. This is the same key used in your frontend snippet.
+> **Note:** Make sure to send API requests to the correct domain. These are `https://app.posthog.com` for US Cloud, `https://eu.posthog.com` for EU Cloud, and your self-hosted domain for self-hosted instances. Confirm yours by checking your URL from your PostHog instance.
 
-# Sending events
+Here are examples of the types of events you can send:
 
-To send events to PostHog, you can use any of [our libraries](/docs/integrate/overview) **or** any Mixpanel library by changing the `api_host` setting to the address of your instance.
+### Single event
 
-If you'd prefer to do the requests yourself, you can send events in the following format:
-
-## Single event
-
-> **Note:** Timestamp is optional. If not set, it'll automatically be set to the current time.
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+  "event": "event name",
+  "api_key": "<ph_project_api_key>",
+  "distinct_id": "user distinct id",
+  "properties": {
+    "total": 500,
```

Comment:
Slightly concnered people wont think that is just an example and will think its required or actually means something
```suggestion
```